src/ssh.c

```diff
@@ -744,6 +759,18 @@ int wolfSSH_stream_exit(WOLFSSH* ssh, int status)
 }
 
 
+int wolfSSH_SendIgnore(WOLFSSH* ssh, const byte* buf, word32 bufSz)
```

Comment:
is the unused buf and bufSz inputs for future use cases?

Reply:
In the spec, the ignore message can be about any size, any data. It is ignored by the peer. The idea is to use random sizes to disguise actual traffic patterns. I haven't seen OpenSSH use it ever. I wanted to update the function at some point to use the parameters.